packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -1124,16 +1221,23 @@ class _InteractiveViewerState extends State<InteractiveViewer> with TickerProvid
       child = LayoutBuilder(
         builder: (BuildContext context, BoxConstraints constraints) {
           final Matrix4 matrix = _transformer.value;
+          final Widget child = widget.builder!(
+            context,
+            _transformViewport(matrix, Offset.zero & constraints.biggest),
+          );
+          if (!widget._transformChild) {
+            return KeyedSubtree(
```

Comment:
I think `_InteractiveViewerBuilt` also conditionally adds an OverflowBox and does clipping: 

https://github.com/flutter/flutter/blob/cc86b703ce5d1f441333df19ad689adeeba14f11/packages/flutter/lib/src/widgets/interactive_viewer.dart#L1178-L1204

So `_constrained` should probably be true.

Reply:
According to how `InteractiveViewer.builder` works it needs to be false:

```dart
assert(!widget.constrained);
```

https://github.com/flutter/flutter/blob/cc86b703ce5d1f441333df19ad689adeeba14f11/packages/flutter/lib/src/widgets/interactive_viewer.dart#L1127

Reply:
According to how `InteractiveViewer.builder` works it needs to be false:

https://github.com/flutter/flutter/blob/cc86b703ce5d1f441333df19ad689adeeba14f11/packages/flutter/lib/src/widgets/interactive_viewer.dart#L1227

Reply:
Added ClipRect / OverflowBox to the child.

But I think it would make sense to do it in `_InteractiveViewerBuilt`.